gen2-record-replay/libraries/depthai_record.py

```diff
@@ -2,8 +2,42 @@
 from pathlib import Path
 from multiprocessing import Process, Queue
 from cv2 import VideoWriter, VideoWriter_fourcc
+import os
 import depthai as dai
 
+def store_frames(path, frame_q):
+    files = {}
+
+    def create_video_file(name):
+        file_path = str(path / f"{name}.mjpeg")
+        # fourcc = VideoWriter_fourcc(*'MJPG')
+        # width = self.nodes[name].getResolutionWidth()
+        # height = self.nodes[name].getResolutionHeight()
+        # writer = VideoWriter(path, fourcc, self.fps, (width, height))
+        # writer.release()
+        # time.sleep(0.001)
+        files[name] = open(file_path, 'wb')
```

Comment:
Could you use [ExitStack](https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack) instead? It's better for handling files

```
with ExitStack() as stack:
    for mgr in ctx_managers:
        stack.enter_context(mgr)
    # ...
```